src/displayCommentPopup.js

```diff
@@ -0,0 +1,165 @@
+  return popRaceCont;
+};
+
+const createPopupNameLabel = (popNameCont) => {
+  const popNameLabel = document.createElement('p');
+  popNameLabel.classList.add('text-white', 'pe-3');
+  popNameLabel.innerText = 'Name:';
+  popNameCont.appendChild(popNameLabel);
+};
+
+const createPopupCardName = (name, popNameCont) => {
+  const popCardName = document.createElement('p');
+  popCardName.classList.add('text-white', 'font-matrix-regular-small-caps', 'font-50px');
+  popCardName.innerHTML = `${name}`;
+  popNameCont.appendChild(popCardName);
+};
+
+const createPopupTypeLabel = (popTypeCont) => {
+  const popTypeLabel = document.createElement('p');
+  popTypeLabel.classList.add('text-white', 'pe-3');
+  popTypeLabel.innerText = 'Type:';
+  popTypeCont.appendChild(popTypeLabel);
+};
+
+const createPopupCardType = (type, popTypeCont) => {
+  const popCardType = document.createElement('p');
+  popCardType.classList.add('text-white', 'font-itc-stone-serif-small-caps-bold', 'font-35px');
+  popCardType.innerHTML = `${type}`;
+  popTypeCont.appendChild(popCardType);
+};
+
+const createPopupCardDescription = (description, popDescCont) => {
+  const popCardDesc = document.createElement('p');
+  popCardDesc.classList.add('text-white', 'font-itc-stone-serif-it-italic', 'font-20px');
+  popCardDesc.innerHTML = `${description}`;
+  popDescCont.appendChild(popCardDesc);
+};
+
+const createPopupRaceLabel = (popRaceCont) => {
+  const popRaceLabel = document.createElement('p');
+  popRaceLabel.classList.add('text-white', 'pe-3');
+  popRaceLabel.innerText = 'Race:';
+  popRaceCont.appendChild(popRaceLabel);
+};
+
+const createPopupCardRace = (race, popRaceCont) => {
+  const popCardRace = document.createElement('p');
+  popCardRace.classList.add('text-white', 'font-matrix-book', 'font-20px');
+  popCardRace.innerHTML = `${race}`;
+  popRaceCont.appendChild(popCardRace);
+};
+
+const displayCommentpopup = (name, type, description, race, image) => {
+  const popOutCont = createPopupOuterContainer();
+  const popInnCont = createPopupInnerContainer(popOutCont);
+  const popImgCont = createPopupImageContainer(popInnCont);
+
+  createPopupCrossIcon(popImgCont);
+  const cardImage = loadImage(image, popImgCont, 'card-image');
+  cardImage.classList.add('y-card');
+
+  createSeparator(popInnCont);
+
+  const popCardInfoCont = createCardInfoContainer(popInnCont);
+
+  const popNameCont = createPopupNameContainer(popCardInfoCont);
+  createPopupNameLabel(popNameCont);
+  createPopupCardName(name, popNameCont);
+
+  const popTypeCont = createPopupTypeContainer(popCardInfoCont);
+  createPopupTypeLabel(popTypeCont);
+  createPopupCardType(type, popTypeCont);
+
+  const popRaceCont = createPopupRaceContainer(popCardInfoCont);
+  createPopupRaceLabel(popRaceCont);
+  createPopupCardRace(race, popRaceCont);
+
+  const popDescCont = createPopupDescriptionContainer(popCardInfoCont);
+  createPopupCardDescription(description, popDescCont);
+};
+
+export default displayCommentpopup;
```

Comment:
Suggestion:
This file has very extensive modularity.
It is arguable that modularity provides better maintenance of code, but in this case, the file is a little too large.
I would suggest reviewing the concept of [DRY](https://www.youtube.com/watch?v=IGH4-ZhfVDk).